tools/scaffolding/scaffolder.go

```diff
@@ -34,6 +34,7 @@ type workflowTemplateValues struct {
 	DeveloperEmail string
 	URLRoot        string
 	URLPath        string
+	WizardTemplate bool
```

Comment:
I think in some cases when we have bools we prepend them with `is`, like IsWizardTemplate

Reply:
Ah good, call, I'll update that